explorer/src/db/persistent_sequence.rs

```diff
@@ -32,6 +32,10 @@ impl<T> PersistentSequence<T> {
     pub fn len(&self) -> u64 {
         self.len
     }
+
+    pub fn is_empty(&self) -> bool {
+        self.len() != 0
```

Comment:
this does the opposite of what I would expect

Reply:
good point, I think is unused though... IIRC clippy complains if you have `len` without `is_empty`

Reply:
Just to be sure, I meant that `self.len() != 0` tells you when it's **not** empty.